### PR TITLE
Made API call for notice to work fine.

### LINKED ISSUE #4: The API called for notices should be working
The changes for this might be needed in `neeto-bugtrap-web`. Keeping this issue here as we might need some changes from the gem too.

@sruthy-j-mallya _a

### PR DESCRIPTION
**Pull request by KrishnapriyaSkk**
Fixes #4 

Changes:
- Updated the route for triggering `notices controller` at neetoBugtrapWeb.
- Removed compression of payload before sending the request to neetoBugtrapWeb.

### CODE REVIEW

**Comment by KrishnapriyaSkk** [comment]
@sandip-mane _a
Please take a look into decoding the recieved request body at bugtrapweb side.

**Comment by KrishnapriyaSkk** [comment]
@sandip-mane _a
Please review.

File: lib/neeto-bugtrap-ruby/util/http.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ def get(endpoint)
       end
 
       def post(endpoint, payload, headers = nil)
-        response = http_connection.post(endpoint, compress(payload.to_json), http_headers(headers))
+        response = http_connection.post(endpoint,  { payload: JSON.parse(payload.to_json).deep_symbolize_keys }.to_json, http_headers(headers))
```
Comment (sandip-mane) [inline lib/neeto-bugtrap-ruby/util/http.rb:47]: 1. Do we need to `deep_symbolize_keys`?
2. Do we need to do to_json and then JSON.parse and then again to_json? I think, without any of this also it should work fine. Or at max, we can do it once.
```suggestion
        response = http_connection.post(endpoint,  { payload: JSON.parse(payload.to_json).deep_symbolize_keys }.to_json, http_headers(headers))
```

**Review by sandip-mane — COMMENTED** [review]
@KrishnapriyaSkk _a please check a few questions.

File: lib/neeto-bugtrap-ruby/util/http.rb (hunk 1 above)
Comment (KrishnapriyaSkk) [reply lib/neeto-bugtrap-ruby/util/http.rb:47]: Yes. I'll remove all those and just keep one `to_json`. Rest will be handled at the BugtrapWeb end.

**Comment by KrishnapriyaSkk** [comment]
@sandip-mane _a
I have made the change. Please review.

**Review by sandip-mane — APPROVED** [review]

**Pull request merged**